apps/stripe/src/modules/app-config/stripe-config.ts

```diff
@@ -0,0 +1,63 @@
+import { err, ok } from "neverthrow";
+
+import { StripePublishableKey } from "@/modules/stripe/stripe-publishable-key";
+import { StripeRestrictedKey } from "@/modules/stripe/stripe-restricted-key";
+
+export class StripeConfig {
+  private constructor(
+    private deps: {
+      name: string;
+      id: string;
+      restrictedKey: StripeRestrictedKey;
+      publishableKey: StripePublishableKey;
+    },
+  ) {}
+
+  static createFromPersistedData(args: {
+    configName: string;
+    configId: string;
+    restrictedKeyValue: string;
+    publishableKeyValue: string;
```

Comment:
I think this is wrong. I would rather put here already valid values (value objects), not validate it here from raw data

Config knows what it needs. It doesn't have to create all of these. 
Your Persistor on the other hand can hold logic how to serialize data. Imagine we have some base64 holding of restricted key. You need to convert it to intermediate "string" just to put string here. Why?

For us, class RestrictedKey is what we *always* should us. No strings, ever, except:
1. Class that saves it, must know how to serialize and parse it. It always receives VO and returns VO. It can save it however it likes, but it can't leak
2. Class that needs raw value (eg stripe sdk) will "getValue" 

except these, there is almost no reason to ever use string